src/components/DrawerMenu.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="drawer-menu__container mx-4 pt-3">
+    <div class="d-flex flex-row justify-content-between">
+      <img src="@/assets/icons/menu-burger.svg" alt="" data-bs-toggle="offcanvas" data-bs-target="#offcanvasRight" aria-controls="offcanvasRight">
+      <Brand />
+    </div>
+    <div class="drawer-menu__offcanvas offcanvas offcanvas-end" tabindex="-1" id="offcanvasRight" aria-labelledby="offcanvasRightLabel">
+      <div class="drawer-menu__items d-flex flex-column align-items-end mx-4 mt-4">
+        <MenuItem title="هلدینگ داده نگار جی" color="#173D7A">
+          <template #icon>
+            <img src="@/assets/icons/left-chevron-dark.svg" alt="" />
+          </template>
+        </MenuItem>
+        <MenuItem title="جی سرور" color="#173D7A">
+          <template #icon>
+            <img src="@/assets/icons/left-chevron-dark.svg" alt="" />
+          </template>
+        </MenuItem>
+        <MenuItem title="وب شات" color="#173D7A">
+          <template #icon>
+            <img src="@/assets/icons/left-chevron-dark.svg" alt="" />
+          </template>
+        </MenuItem>
+        <MenuItem title="JeoIP" color="#173D7A">
+          <template #icon>
+            <img src="@/assets/icons/left-chevron-dark.svg" alt="" />
+          </template>
+        </MenuItem>
+        <div class="drawer-menu__divider"></div>
+        <MenuItem title="پشتیبانی" color="#173D7A">
```

Comment:
link it to https://jeyserver.com/fa/userpanel/ticketing

Reply:
used href on MenuItem component